KIPs/kip-15.md

```diff
@@ -0,0 +1,136 @@
+---
+kip: 15
+title: Additional LM on ETH & Polygon
+status: Active
+type: Spend KNC
+authors: Kyber Network team
+created: 2021-09-06
+---
+
+```
+executor: short timelock executor
+voting_power: current staked voting power
+execution: transfer 95,350 knc to 0xD69D7A1031e6E63a162414F9A77278757690C30E (kyber multisig to handle KNC bridging between ethereum and other chains)
```

Comment:
The execution should depend on how many projects are accepted?

Reply:
Is it okay if we say "transfer knc to 0xD69D7A1031e6E63a162414F9A77278757690C30E, depending on projects accepted"